dev/test-dependencies.sh

```diff
@@ -119,7 +119,7 @@ for HADOOP_HIVE_PROFILE in "${HADOOP_HIVE_PROFILES[@]}"; do
     }' | sort | grep -v spark > dev/pr-deps/spark-deps-$HADOOP_HIVE_PROFILE
 done
 
-if [[ $@ == **replace-manifest** ]]; then
+if [[ "$*" == **replace-manifest** ]]; then
```

Comment:
SC2199 (error): Arrays implicitly concatenate in [[ ]]. Use a loop (or explicit * instead of @).